data-raw/state_census.R

```diff
@@ -1,4 +1,7 @@
-state_census <- covidcast::state_census %>%
+library(dplyr)
+library(covidcast)
+
+state_census <- state_census %>%
```

Comment:
nitpick: maybe keep `covidcast::state_census` on the RHS to clarify RHS is there and LHS will be in global environment?